core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -294,6 +294,9 @@ public enum Property {
   GENERAL_DELEGATION_TOKEN_UPDATE_INTERVAL("general.delegation.token.update.interval", "1d",
       PropertyType.TIMEDURATION, "The length of time between generation of new secret keys.",
       "1.7.0"),
+  GENERAL_IDLE_PROCESS_INTERVAL("general.metrics.process.idle", "5m", PropertyType.TIMEDURATION,
+      "Amount of time a process must be idle before the accumulo.server.idle metric is incremented.",
```

Comment:
Do you think it would be better to make it more general in case things change again in the future? Something like "the amount of time ... before it is considered to be idle by the metrics system"

Reply:
Yeah, that's fine too.